build.sbt

```diff
@@ -14,11 +14,11 @@ ThisBuild / scmInfo := Some(
 ThisBuild / startYear := Some(2020)
 Global / excludeLintKeys += scmInfo
 
-val Scala213 = "2.13.4"
+val Scala213 = "2.13.6"
 ThisBuild / spiewakMainBranches := Seq("main")
 
-ThisBuild / crossScalaVersions := Seq(Scala213, "3.0.0-RC2", "2.12.10")
-ThisBuild / versionIntroduced := Map("3.0.0-RC1" -> "3.0.0")
+ThisBuild / crossScalaVersions := Seq(Scala213, "3.0.0", "2.12.14")
+ThisBuild / versionIntroduced := Map("3.0.0" -> "3.0.0")
```

Comment:
this should probably be `3.0.0 -> 0.3.0` (or whatever the next version is going to be) now that I think about it